### PR TITLE
Monitor Capacitor applications guide Browser SDK

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do? What is the motivation?

- Adds a new guide on how to monitor Capacitor applications with the Browser SDK, as per [DOCS-8518](https://datadoghq.atlassian.net/browse/DOCS-8518).

### Merge instructions

- [ ] Please merge after reviewing

### Additional notes

[DOCS-8518]: https://datadoghq.atlassian.net/browse/DOCS-8518?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by rtrieu** [comment]
Hi @nhientrandatadog and @DataDog/rum-mobile, can you review this to make sure it's accurate and complete? In particular, the first known limitation (troubleshooting) seems like a big limitation - should it be addressed in the intro?

**Comment by nhientrandatadog** [comment]
@rtrieu yes, let's add something in the first section, maybe inline with:

You can install and configure the Datadog Browser SDK to start monitoring applications you’ve built with Capacitor. This will provide you with visibility into the JavaScript portion of your application; **native application visibility is not supported with this workaround**.

**Review by BenoitZugmeyer — APPROVED** [review]
Looks good to me though I am unfamiliar with Capacitor

**Comment by MaelNamNam** [comment]
LGTM, thanks a lot!

File: content/en/real_user_monitoring/guide/monitor-capacitor-applications-using-browser-sdk.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+description: Learn how to monitor cross-platform applications built with Capacitor through the browser RUM SDK.
+further_reading:
+  - link: '/real_user_monitoring/'
+    tag: 'Documentation'
+    text: 'Learn about Real User Monitoring'
+---
+
+## Overview
+
+[Capacitor][1] is an open source native runtime for building Web Native applications that run natively on iOS, Android, and progressive web applications with JavaScript, HTML, and CSS.
```
Comment (evorotnikov-dd) [inline content/en/real_user_monitoring/guide/monitor-capacitor-applications-using-browser-sdk.md:12]: I think there is too much `native` words here which make it confusing a bit. IS that a default capacitor definition from their website?

File: content/en/real_user_monitoring/guide/monitor-capacitor-applications-using-browser-sdk.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+You can install and configure the Datadog Browser SDK to start monitoring applications you've built with Capacitor. The configuration provides you with visibility into the JavaScript portion of your application (excludes native application visibility).
```
Comment (evorotnikov-dd) [inline content/en/real_user_monitoring/guide/monitor-capacitor-applications-using-browser-sdk.md:14]: i woudl rpobably clarify what is "(excludes native application visibility)" is, maybe as "(excludes native application visibility e.g. native app startup time, native crashes etc.)"

File: content/en/real_user_monitoring/guide/monitor-capacitor-applications-using-browser-sdk.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+   });
+   ```
+
+3. Once you've configured the SDK correctly, your data populates the [RUM Explorer][3].
+
+## Troubleshooting
+
+### I only have visibility on the JavaScript portion of my application, but not the native part
+
+This is expected behavior. The native part of a Capacitor application, whether it is through the usage of plugins or custom code, is not monitored. Usually plugins forward a response status that can be tracked from the JavaScript side of the application. However, if a plugin crashes, or the entire application crashes due to native code issues, it is not reported to Datadog.
```
Comment (evorotnikov-dd) [inline content/en/real_user_monitoring/guide/monitor-capacitor-applications-using-browser-sdk.md:43]: So i saw client that instrument both native and capacitor separately, they pay double however they samples native at a very low rate just to see some instances of crashes etc. Should this be added here? We may want to consult with eng about it.

File: content/en/real_user_monitoring/guide/monitor-capacitor-applications-using-browser-sdk.md (hunk 1 above)
Comment (rtrieu) [reply content/en/real_user_monitoring/guide/monitor-capacitor-applications-using-browser-sdk.md:12]: Hi @evorotnikov-dd on their [website](https://capacitorjs.com/) they describe themselves like so:

A cross-platform native runtime for web apps.
Capacitor is an open source native runtime for building Web Native apps. Create cross-platform iOS, Android, and Progressive Web Apps with JavaScript, HTML, and CSS.

Would this be better?

```suggestion
[Capacitor][1] is an open source native runtime for building Web Native applications that run on iOS, Android, and progressive web applications with JavaScript, HTML, and CSS.
```

File: content/en/real_user_monitoring/guide/monitor-capacitor-applications-using-browser-sdk.md (hunk 3 above)
Comment (rtrieu) [reply content/en/real_user_monitoring/guide/monitor-capacitor-applications-using-browser-sdk.md:43]: cc @nhientrandatadog @BenoitZugmeyer @MaelNamNam Can you provide guidance here?

**Pull request merged**